.bit/responses/1.4-Week 1 Step 4.md

```diff
@@ -244,14 +223,18 @@ module.exports = function (context, req) {
     context.done();
 };
 ```
-Request parameters are a property of the `req` or request parameter of the module. The request has a `query` object that stores all of the parameters passed in. You would access a parameter by calling on the query like this:
+Request parameters are a property of the `req` or request parameter of the module. The request has a `query` object that stores all of the parameters passed in. You don't need to specify what parameters the user needs to input into the HTTP request, but can acess any parameters that are sent in. You would access a parameter by calling on the query like this:
```

Comment:
I'm not sure if I'm mistaken, but I'm pretty sure you have to specify the parameter the user uses in order for you to access it.

Reply:
No, you don't need to specify what the user can input in as a parameter. Ie they can enter in '?name=fifi' into the HTTP request without breaking anything, we just don't use it.

Reply:
@FifiTeklemedhin Oh that makes sense - thanks for clarifying!